src/json-api/json-api-module.ts

```diff
@@ -391,4 +393,8 @@ export class JsonApiModule {
     };
   }
 
+  public toApiNotificationConfiguration(configuration: NotificationConfiguration): ApiNotificationConfiguration {
+    return configuration as ApiNotificationConfiguration;
```

Comment:
This function seems a bit suspect. The function name makes it seem that some sort of conversion is being done when, in reality, it's just hard-casting it to another type.